### ISSUE TITLE
Only set `latest` Docker tag on official releases

### ISSUE THREAD

**Issue by wslulciuc** [MarquezProject/marquez#1523]

**Comment by deedubs** [comment]
You may want to consider not using `latest` at all, and encourage people to select a specific release.  While a best practice for using containers in production, I think it'll also help you when it comes to helping community members as you'll be able to concretely know which version of Marquez they are having issues with.

Just my .02

(I'm only just beginning to poke around marquez, great work and thank you)

**Comment by wslulciuc** [comment]
@deedubs: I completely agree. We've been considering removing the `latest` tag and replace it with the `snapshot` version for Marquez. Which, _techincally_, can be viewed as the _latest_ and greatest.
